src/core/builders/create-form-validator.ts

```diff
@@ -116,21 +119,35 @@ export const createFormValidator = <Values extends object, Err>(
   return formValidator;
 };
 
-const validate: ValidateFn = config => ({
-  type: "field",
-  field: config.field,
-  triggers: config.triggers,
-  validators: config.rules,
-  dependencies: config.dependencies,
-});
-
-validate.each = config => ({
-  type: "each",
-  field: config.field as any,
-  triggers: config.triggers,
-  validators: config.rules,
-  dependencies: config.dependencies,
-});
+const validate: ValidateFn = <T, Err, Deps extends any[]>(
+  x: ValidateConfig<T, Err, Deps> | ValidateField<T, Err>,
+  ...rules: Array<Validator<T, Err>>
+): FieldValidator<T, Err, Deps> => {
+  const config: ValidateConfig<T, Err, Deps> =
+    (x as any)["field"] != null
```

Comment:
maybe `x["__path"] != null` and reverse logic fits better @mixvar ?

Reply:
I think its ok as is